docker-files/Dockerfile.ubi.micro

```diff
@@ -17,7 +17,7 @@
 # Dockerfile to build PowerStore CSI Driver
 # based on UBI-micro image
 # Requires: RHEL host with subscription
-# UBI Image: ubi8/ubi-micro:8.8-1
+# UBI Image: ubi9/ubi-micro:9.2-9
```

Comment:
Should we use 9.2-13?

From [Redhat ](https://catalog.redhat.com/software/containers/ubi9/ubi-micro/615bdf943f6014fa45ae1b58?tag=9.2-9&push_date=1687362802000&container-tabs=gti&gti-tabs=unauthenticated) website:
alert:Updated image available
Red Hat strongly recommends updating to the newest image version 9.2-13 unless otherwise defined by the support policy of Red Hat

Reply:
done